lib/Helper/MergeConfig.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from 'fs';
+
+// merges the config files
+export function mergeConfigFile(
+  sourcePath: string,
+  templatePath: string,
+): boolean {
+  try {
+    const templateFile = fs.readFileSync(templatePath, 'utf8');
+    const sourceFile = fs.readFileSync(sourcePath, 'utf8');
+    const newText = templateFile.replace('// ORIGINAL CONFIG', sourceFile);
+    new Function(newText); // check if the file is valid javascript
```

Comment:
I don't think this will throw if you pass in invalid JS. I would just remove this line.

Reply:
Ah you're right. I messed up how I verified this. Sorry about that!